railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -104,6 +105,10 @@ def config_when_updating
       unless active_record_belongs_to_required_by_default_config_exist
         remove_file 'config/initializers/active_record_belongs_to_required_by_default.rb'
       end
+
+      unless active_record_destroy_all_in_batches_config_exist
+        remove_file 'config/initializers/active_record_destroy_all_in_batches.rb'
```

Comment:
We should put this in the `config/initializers/new_framework_defaults.rb` instead.